src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LanguageClient/ProjectSystem/RazorProjectInfoEndpointPublisher.cs

```diff
@@ -121,7 +122,8 @@ private void ProjectManager_Changed(object sender, ProjectChangeEventArgs args)
         }
     }
 
-    private void EnqueuePublish(IProjectSnapshot projectSnapshot)
+    // protected for tests
+    protected void EnqueuePublish(IProjectSnapshot projectSnapshot)
```

Comment:
Same feedback as above. Both of these methods can remain private if you provide a test accessor.